src/ReplicatedStorage/Client/InstanceComponents/Misc/CursorLook.lua

```diff
@@ -40,10 +41,12 @@ local function clampedAngles(angles: Vector3, lowerBounds: Vector3, upperBounds:
 end
 
 function CursorLook.new(instance: Instance)
+	local clientComm = Comm.ClientComm.new(instance, false, "Cursor")
 	local self = setmetatable({}, CursorLook)
```

Comment:
Try not to create things before creating self - it makes this function half method half static and it's a bit confusing if it was bigger.